docarray/typing/tensor/abstract_tensor.py

```diff
@@ -71,6 +83,7 @@ class AbstractTensor(Generic[TTensor, T], AbstractType, ABC):
 
     __parametrized_meta__: type = _ParametrizedMeta
     _PROTO_FIELD_NAME: str
+    __unparametrizedcls__: Optional[type] = None
```

Comment:
Couldn't this just be `__docarray_target_shape__` so that we don't need a separate class for paremtrized tensors anymore? And if there is no shape given, this is `None`?

Reply:
We still need a separate class for parametrized tensors right? Otherwise they cant be treated as types?